iModelCore/Bentley/BeRapidJson/PublicAPI/BeRapidJson/BeJsValue.h

```diff
@@ -271,16 +273,9 @@ struct BeJsConst {
             Utf8String::Sscanf_safe(str, fmt, &val);
             return val;
         }
-        if (isNumeric()) {
-            // Casting a double to a uint produces undefined results while casting a double to int is well defined.
-            // Similarly, casting int to uint is well defined.
-            // On ARM casting a negative double to uint results in 0.
-            // On Intel, casting a negative double to uint produces the same result as below.
-            return uint64_t(int64_t(GetDouble((double)defVal)));

```

Comment:
Little unsure about getting rid of this entire snippet outright.. Should this potentially be part of the implementation in napi.h of Getuint64?

Reply:
Yes, that's the proposal: let the napi implementation deal with the JS precision constraints, don't impose them on the other implementations.

Reply:
Moved the snippet over to GetUint64 in napi.h